homeassistant/components/switch_as_x/cover.py

```diff
@@ -87,4 +89,7 @@ def async_state_changed_listener(
         ):
             return
 
-        self._attr_is_closed = state.state != STATE_ON
+        if not self._invert_state:
```

Comment:
I'd put the default as the else case and the special case as the if case.

Reply:
Fixed 👍